models/classes/runner/time/TimerAdjustmentService.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA ;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiTest\models\runner\time;
+
+use oat\oatbox\service\ConfigurableService;
+use oat\taoDelivery\model\execution\DeliveryExecutionInterface;
+use oat\taoQtiTest\models\runner\StorageManager;
+use oat\taoQtiTest\models\TestSessionService;
+use oat\taoTests\models\runner\time\TimerAdjustmentServiceInterface;
+use qtism\data\QtiIdentifiable;
+
+class TimerAdjustmentService extends ConfigurableService implements TimerAdjustmentServiceInterface
+{
+    /** @var DeliveryExecutionInterface */
+    private $deliveryExecution;
+
+    /**
+     * @inheritDoc
+     */
+    public function increase(
+        DeliveryExecutionInterface $deliveryExecution,
+        int $seconds,
+        QtiIdentifiable $source = null
+    ): bool {
+        $this->deliveryExecution = $deliveryExecution;
```

Comment:
As an option probably it would be better to use TestSession as a parameter since we do not use DeliveryExecution in any other way except to get test session but it's not something critical.

Reply:
Done